src/pages/Store/StorePage/components/EventCarousel/index.tsx

```diff
@@ -65,112 +55,106 @@ function Card({
         </div>
         <div className={styles['swipe-item__nowrap']}>할인 혜택을 받아보세요!</div>
       </div>
-    </Link>
+    </button>
   );
 }
 
-function EventCarousel() {
-  const carouselList = useGetAllEvents();
+export default function EventCarousel() {
   const isMobile = useMediaQuery();
-  const {
-    emblaRef, canNextClick, canPrevClick, currentIndex, scrollTo,
-  } = useCarouselController(isMobile);
-  const logger = useLogger();
-  const eventLogging = (shopName: string) => {
-    logger.actionEventClick({
-      actionTitle: 'BUSINESS', event_label: 'shop_categories_event', value: `${shopName}`, event_category: 'click',
-    });
+  const { events } = useGetAllEvents();
+  const { emblaRef, currentIndex, scrollTo } = useCarouselController(isMobile);
+
+  if (events.length < 1) return null;
```

Comment:
events 개수가 없다면 여기서 따로 처리해도 괜찮을 거 같은데, 다른 곳에서 처리하고 있나요?

Reply:
이미 머지했지만... `events`가 없을 때 사용자에게 UI를 보여줘야 할 거 같아요. 그걸 return null로 처리하고 있는데 아래 `DUMMY_EVENT`와 비슷하게 빈 UI를 보여주는 게 좋지 않을까 생각했어요!

```suggestion
  if (events.length < 1) return <EmptyEvent />
```

이런식으로 값이 없을 때 컴포넌트를 보여주는게 좋을 거 같다고 생각했어요!